### PR TITLE
isp-imx: start_isp.sh: fix NR_DEVICE_TREE_BASLER variable

### PR DESCRIPTION
**Pull request by chrisdimich**
Also remove uneeded -lm option from run.sh script invocation.

### CODE REVIEW

**Comment by otavio** [comment]
@chrisdimich @angolini

**Comment by chrisdimich** [comment]
Shall I manually backport?

File: recipes-bsp/isp-imx/isp-imx_4.2.2.13.0.bb [hunk 1]
```diff
@@ -7,7 +7,7 @@ DEPENDS = "python3 libdrm virtual/libg2d"
 
 SRC_URI = " \
     ${FSL_MIRROR}/${BP}.bin;fsl-eula=true \
-    file://0001-start_isp.sh-fix-test-to-be-generic.patch \
+    file://0001-start_isp.sh-fix-NR_DEVICE_TREE_BASLER-variable.patch \
```
Comment (thochstein) [inline recipes-bsp/isp-imx/isp-imx_4.2.2.13.0.bb:10]: The patch name should not change.

Comment (thochstein) [reply recipes-bsp/isp-imx/isp-imx_4.2.2.13.0.bb:10]: Oops, sorry, never mind, I see what you did.

**Comment by chrisdimich** [comment]
So shall I manually backport to hardknott? We need it there too.

**Comment by thochstein** [comment]
I got it: #869

**Pull request merged**